main.go

```diff
@@ -40,6 +40,7 @@ var cfg struct {
 	BuildKitAutoDiscoveryKubernetesPodSelector                string
 	BuildKitAutoDiscoveryKubernetesNamespace                  string
 	BuildKitAutoDiscoveryKubernetesLeasePrefix                string
+	BuildKitAutoDiscoveryScaleStatefulset                     string
```

Comment:
Just a suggestion, but should we just call this `BuildKitAutoDiscoveryStatefulset`?